### PR TITLE
CkWriter Changes

### PR DESCRIPTION
**Pull request by acpaquette**
Updates ckwriter to handle loading CK kernels without using the Spice class.

## Description
Removes ckwriters dependency on the Spice class for loading some of the kernels it needs to actually write the new ck kernel.

## Related Issue
None

## Motivation and Context
Merging ale into ISIS for ISIS4 release.

## How Has This Been Tested?
CkWriter tests.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/base/apps/ckwriter/CkSpiceSegment.cpp [hunk 1]
```diff
@@ -222,12 +222,9 @@ void CkSpiceSegment::import(Cube &cube, const QString &tblname) {
...
 
     // Load necessary kernels (IAK for Cassini, mainly)
-    _kernels.Load("FK,SCLK,LSK,IAK");
-
-//    _kernels.Load("FK,SCLK,LSK,IAK,SPK");
-//   QStringList loaded = _kernels.getLoadedList();
-//   cout << "\nKernels prior to conversion...\n" << loaded.join("\n");
+    _kernels.Load("CK,FK,SCLK,LSK,IAK");
 
+    QStringList loaded = _kernels.getLoadedList();
```
Comment (jessemapel) [inline isis/src/base/apps/ckwriter/CkSpiceSegment.cpp]: Where is this used?

**Review by jessemapel — APPROVED** [review]
Looks good just one question

File: isis/src/base/apps/ckwriter/CkSpiceSegment.cpp (hunk 1 above)
Comment (acpaquette) [reply isis/src/base/apps/ckwriter/CkSpiceSegment.cpp]: It gets used in a commented out print, I'll remove the variable

**Review by krlberry — APPROVED** [review]
:+1:

**Pull request merged**